docs/docs/docker-compose-setup.md

```
@@ -312,71 +332,79 @@ python -c "import base64; import os; print(base64.urlsafe_b64encode(os.urandom(3
-docker compose -f docker-compose-with-mongodb.yml config
+# Legacy v1 commands (replace "docker compose" with "docker-compose"):
+docker-compose -f docker-compose.yml up -d
+docker-compose -f docker-compose-with-mongodb.yml up -d
+docker-compose -f docker-compose-with-mongodb.yml logs -f
+docker-compose -f docker-compose-with-mongodb.yml down
+```
 
-# Pull all required images
-docker compose -f docker-compose-with-mongodb.yml pull
+**Creating an alias** (optional): To use the same commands as shown in this guide, create an alias:
+```bash
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)